common/teacmd/editor.go

```diff
@@ -65,10 +74,6 @@ func SetupCardinalEditor() error {
 
 	downloadURL := latestReleaseURL
 	downloadVersion, versionIsExist := cardinalVersionMap[cardinalVersion]
-	if !versionIsExist {
```

Comment:
We still need this check, as the cardinal-editor repo can be out-of-date. This could happen if there is a new release of cardinal but the tag is not updated in the `version_map.json` file. Anytime the cardinal version isn't in the version map, it should default to the latest version.

Reply:
whoops, yeah missed this earlier :sweat_smile: